stream/stream.h

```diff
@@ -32,6 +32,12 @@
 // it's guaranteed that you can seek back by <= of this size again.
 #define STREAM_BUFFER_SIZE 2048
 
+// Size of 0 byte padding. The padding is not included in the returned length.
+#define STREAM_PADDING_SIZE 1
```

Comment:
the way it is now it mistakenly looks like this padding applies to all stream operations, but it's only for stream_read_complete

personally I would use `2140000000` for `STREAM_MAX_READ_SIZE` and just put the hardcoded 1 byte padding back into the function

Reply:
Removed this define and instead hardcoded 1 below.